src/vcBPA.cpp

```diff
@@ -1008,6 +1039,15 @@ vdkError vcBPA_ConvertReadPoints(vdkConvertCustomItem *pConvertInput, vdkPointBu
     float *pDisplacement = (float*)udAddBytes(pBuffer->pAttributes, pointAttrOffset + displacementOffset);
     *pDisplacement = (float)distance;
 
+    for (int elementIndex = 0; elementIndex < 3; ++elementIndex) //X,Y,Z
```

Comment:
Could use `trianglePoint::ElementCount` or `udDouble3::ElementCount` instead of `3` - your call, with the comment it makes sense either way.